recipe/meta.yaml

```diff
@@ -124,16 +128,17 @@ outputs:
       noarch: generic
     requirements:
       run:
-        - cdo >=1.9.7
         - {{ pin_subpackage('esmvaltool-python', exact=True) }}
+        - ncl
+        - cdo
```

Comment:
I would keep the pins, are they missing from environment.yml?

Reply:
Yes: [environment.yml](https://github.com/ESMValGroup/ESMValTool/blob/93d6ff87e3f7b3adc6d2425ae826658928585f38/environment.yml#L99)

Reply:
I think it's academic now, cdo is at 2.4.1, but why not, only God knows what bizarre solution mamba finds with cdo <1.9.7

Reply:
I made this change: c027a0c bsed on the fact that I pinned cdo to `>=2.3.0` via https://github.com/ESMValGroup/ESMValTool/pull/3611 😊